libs/vm/include/vm/object.hpp

```diff
@@ -209,6 +221,11 @@ class Object
   virtual bool SerializeTo(ByteArrayBuffer &buffer);
   virtual bool DeserializeFrom(ByteArrayBuffer &buffer);
 
+  TypeId getTypeId() const
```

Comment:
Style: `type_id()`

Reply:
This is GetTypeId() elsewhere in the code